backend/src/main/scala/cromwell/backend/callcaching/CacheHitDuplicating.scala

```diff
@@ -43,75 +52,49 @@ trait DuplicatingCacheHitCopyingActor extends BackendCacheHitCopyingActor {
     )
   }
 
+  /**
+    * After copying files, return the simpletons substituting the destination file paths.
+    */
   private def copySimpletons(wdlValueSimpletons: Seq[WdlValueSimpleton],
```

Comment:
ToL: Presumably if these fail we let-it-crash(tm). Could you double-check the mixing-in actors are properly supervised? (and then tell me how to find out!)